server/api/person/person.email.js

```diff
@@ -21,8 +22,8 @@ module.exports.emailPerson = async (template, to, props, renderOnly = false) =>
     send: true,
     subjectPrefix: config.env === 'production' ? false : `[${config.env.toUpperCase()}] `,
 
-    // Comment the line below to see preview email
-    preview: false,
+    // uncomment the line below to see preview email
+    // preview: true,
```

Comment:
Is leaving `preview` commented out equivalent to having `preview: false`?

Reply:
it is